src/main/resources/hudson/security/table.js

```diff
@@ -201,9 +204,16 @@ Behaviour.specify(".global-matrix-authorization-strategy-table TD.stop A.migrate
   e = null; // avoid memory leak
 });
 
+/*
+ * Determine which attribute to set tooltips in. Changed in Jenkins 2.379 with Tippy and data-html-tooltip support.
+ */
 function getTooltipAttributeName() {
-  var tippySupported = window.registerTooltips !== undefined;
-  return tippySupported ? 'html-tooltip' : 'tooltip';
+  let coreVersion = document.body.getAttribute('data-version');
+  if (coreVersion === null) {
+    return 'tooltip'
+  }
+  let tippySupported = coreVersion >= '2.379';
```

Comment:
I would think about adding a TODO, this check could possibly break with interesting version numbers
```suggestion
  // TODO remove after minimum version is 2.379 or higher
  let tippySupported = coreVersion >= '2.379';
```

Reply:
If we get to 2.1000, yes 😁

Reply:
heh yeah that was the only one I found it broke on I did try quite a few in the browser

Reply:
It's an ascii-left-to-right string comparison. If we limit it to version numbers, the sort order is like in https://stats.jenkins.io/plugin-installation-trend/installations.json